docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml

```diff
@@ -185,8 +185,9 @@
               <row>
                 <entry>-main <emphasis>&lt;ref&gt;</emphasis></entry>
 
-                <entry>Compile definition &lt;ref&gt; from the source
-                collection</entry>
+                <entry>Compile definition &lt;ref&gt; from the specified
+                source - note extended github repository support (see "Git
```

Comment:
Is it only GitHub or does this work with any Git repository? If you are going to leave GitHub in the sentence, then it should use the capitalization they use--GitHub

Reply:
I believe works with any Git - GitHub or GitLab, however perhaps @ghalliday can confirm?

Reply:
It can work with github or gitlab, but it defaults to using a prefix that will use github

Reply:
@JamesDeFabia note the <link> for the see also in the main option table entry does in fact generate working link : http://10.224.20.18/view/Docs-gp/job/DocBuild-01-GPRepo/65/artifact/build-docs/Release/docs/EN_US/TheECLIDEandHPCCClientTools_EN_US-9.0.7-closedown0.pdf#page=115